p2p/conn/secret_connection.go

```diff
@@ -140,12 +147,14 @@ func MakeSecretConnection(conn io.ReadWriteCloser, locPrivKey crypto.PrivKey) (*
 	}
 
 	sc := &SecretConnection{
-		conn:       conn,
```

Comment:
The previous comment applies to here: keep `conn` as underlying connection, please.